### PR TITLE
WIP Renaming topics mohamed

### PR DESCRIPTION
**Pull request by moh-abdelhady04**
## Related Issue(s)
(https://github.com/mcgill-robotics/AUV-2024/issues/218)

## Other Dependent PRs
  - auv-sim-gazebo PR
  - auv-sim-unity PR

## Changes Made
- Renamed the topics as per instructions. All details mentioned in commit messages.

## Required Wiki Changes (if any)
- Update any topic names that are on the wiki

### CODE REVIEW

File: catkin_ws/src/controls/src/superimposer.py [hunk 1]
```diff
@@ -11,8 +11,8 @@
 class Superimposer:
     def __init__(self):
         # forces in robot reference frame
-        self.surge = Superimposer.Degree_Of_Freedom('surge')
-        self.sway = Superimposer.Degree_Of_Freedom('sway')
+        self.surge = Superimposer.Degree_Of_Freedom('/controls/force/surge')
+        self.sway = Superimposer.Degree_Of_Freedom('/controls/force/sway')
         self.heave = Superimposer.Degree_Of_Freedom('heave')
         self.roll = Superimposer.Degree_Of_Freedom('roll')
         self.pitch = Superimposer.Degree_Of_Freedom('pitch')
```
Comment (frompotenza) [inline catkin_ws/src/controls/src/superimposer.py]: what about heave, roll, and pitch?

**Comment by frompotenza** [comment]
Have you done any tests?

**Review by frompotenza — CHANGES_REQUESTED** [review]

File: catkin_ws/src/controls/src/servers/base_server.py [hunk 2]
```diff
@@ -28,21 +28,21 @@ def __init__(self) -> None:
...
     def establish_effort_publishers(self):
-        self.pub_surge = rospy.Publisher('surge', Float64, queue_size=1)
-        self.pub_sway = rospy.Publisher('sway', Float64, queue_size=1)
-        self.pub_heave = rospy.Publisher('heave', Float64, queue_size=1)
-        self.pub_roll = rospy.Publisher('roll', Float64, queue_size=1)
+        self.pub_surge = rospy.Publisher('/controls/force/surge', Float64, queue_size=1)
+        self.pub_sway = rospy.Publisher('/control/force/sway', Float64, queue_size=1)
+        self.pub_heave = rospy.Publisher('/controls/force/heave', Float64, queue_size=1)
+        self.pub_roll = rospy.Publisher('/controls/torque/roll', Float64, queue_size=1)
         self.pub_pitch = rospy.Publisher('pitch', Float64, queue_size=1)
```
Comment (antoinedang) [inline catkin_ws/src/controls/src/servers/base_server.py]: missed pitch here

**Comment by antoinedang** [comment]
@moh-abdelhady04 could you link to the list of all the topics you changed? Would make reviewing this easier

**Comment by frompotenza** [comment]
For some topics, it's like state_x and for others is state/x (this is just an example, not that there is a topic named state_x). Is that on purpose?

**Comment by antoinedang** [comment]
tests complete: additional integration tests (pending PR but tested locally), run sim + check all topic names that show up

passed all of them so I'm gonna merge

**Pull request merged**